geode-core/src/main/java/org/apache/geode/distributed/internal/ClusterDistributionManager.java

```diff
@@ -1793,9 +1793,7 @@ private String prettifyReason(String r) {
    */
   private void handleManagerStartup(InternalDistributedMember theId) {
     // Note test is under membersLock
-    if (theId.getVmKind() != ClusterDistributionManager.LOCATOR_DM_TYPE) {
-      stats.incNodes(1);
-    }
+    stats.incNodes(1);
```

Comment:
I think @Bill @bschuchardt or @dschneider-pivotal should review this change instead of me.

Reply:
Why is it ok to remove the if-condition from here?

Reply:
The stats should count both locators and servers, as both of them are nodes. With if-condition locators wouldn't count.